parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -259,17 +271,31 @@ private WriteBuilder createContextFunc(Function<Map<String, String>, Context> ne
           conf.set(entry.getKey(), entry.getValue());
         }
 
-        ParquetProperties parquetProperties = ParquetProperties.builder()
+        ParquetProperties.Builder propsBuilder = ParquetProperties.builder()
             .withWriterVersion(writerVersion)
             .withPageSize(pageSize)
             .withDictionaryPageSize(dictionaryPageSize)
-            .build();
+            .withMaxBloomFilterBytes(bloomFilterMaxBytes)
+            .withBloomFilterEnabled(bloomFilterEnabled);
+
+        new ColumnConfigParser()
```

Comment:
looks a bit weird, is this intended, new instance without reference to it ?

Reply:
This ColumnConfigParser instance is created to parse the config and apply to propsBuilder

Reply:
I know it's probably not a big deal, but I mean could new instances be avoided ?